ambry-tools/src/test/java/com.github.ambry/store/MockStoreKeyConverter.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2018 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.store;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+
+
+/**
+ * Mock StoreKeyConverter, basically a wrapper over a hashmap.
+ * Not found StoreKeys are returned with the value == key
+ */
+class MockStoreKeyConverter implements StoreKeyConverter {
```

Comment:
@zzmao  has already added one - its in ambry-api. If you want it to return same key on a miss (as opposed to null), i suggest adding a config to that code to make it so.

Reply:
config i.e an option in the constructor

Reply:
added